Rmarkdown/Multiple_Ascending_Dose_PKPD_ordinal.Rmd

```diff
@@ -195,10 +195,8 @@ gg <- gg + ylab(conc_label)
 gg
 ```
 
-
 ## Explore variability
```

Comment:
It's a little bit weird that the x-ticks don't go all the way up to 9 and instead stop at 8.  Should they go up to 9?